File: 3588/3588.c
int findMaximumAreaOfATriangle(struct TreeNode* root) {
    if (!root) return 0;
    int l = findMaximumAreaOfATriangle(root->left);
    int r = findMaximumAreaOfATriangle(root->right);
    return 1 + (l > r ? l : r);
}

File: 3588/3588.py
class Solution:
    def find_maximum_area_of_a_triangle(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.find_maximum_area_of_a_triangle(root.left), self.find_maximum_area_of_a_triangle(root.right))